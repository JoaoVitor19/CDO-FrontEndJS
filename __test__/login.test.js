/* global __DEV__ */
import React from 'react'
import { render } from '@testing-library/react-native'
import Relatorio from '../screens/Relatorios/Relatorios'
// const {Relatorio} = require('../screens/Relatorios/Relatorios')

jest.mock('../screens/Relatorios/Relatorios', () => '../screens/Relatorios/Relatorios');


describe('Login', () => {
    test('should render', () => {
        const navigation = { navigate: () => { } }
        const { toJSON } = render(<Relatorio />)

        expect(toJSON()).toMatchSnapshot()
    })
})

// it('works', () => {
//     expect(1).toBe(1);
//   });