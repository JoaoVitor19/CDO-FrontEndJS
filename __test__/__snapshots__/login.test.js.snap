// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should render 1`] = `<../screens/Relatorios/Relatorios />`;
